synapse/storage/data_stores/main/events_worker.py

```diff
@@ -1082,6 +1082,21 @@ async def get_all_new_backfill_event_rows(
     ) -> Tuple[List[Tuple[int, list]], int, bool]:
         """Get updates for backfill replication stream, including all new
         backfilled events and events that have gone from being outliers to not.
+
+        Args:
+            instance_name: The writer we want to fetch updates from. Unused
+                here sincethere is only ever one writer.
+            last_id: The token to fetch updates from. Exclusive.
+            current_id: The token to fetch updates up to. Inclusive.
+            limit: The requested limit for the number of rows to return. The
+                function may return more or fewer rows.
+
+        Returns:
+            A tuple consisting of: the updates, the position of the rows
+            returned up to, and whether we returned fewer rows than exists
```

Comment:
can you rephrase "the position of the rows returned up to"? it's somewhat unclear: inconsistent use of "position" and "token", inclusive or exclusive, etc.

"the last token included in the results", maybe?

Reply:
Hopefully I've clarified it. Unfortunately, technically, the returned token doesn't *have* to be the last token included in the results (since `current_id` could be greater due to a gap)